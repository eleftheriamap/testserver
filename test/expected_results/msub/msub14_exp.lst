
test/exp/test/gen/msub/msub14_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	91781400 	add	x0, x0, #0xe05, lsl #12
   4:	d1103fbd 	sub	x29, x29, #0x40f
   8:	516e5a10 	sub	w16, w16, #0xb96, lsl #12
   c:	1b1dc006 	msub	w6, w0, w29, w16
  10:	8a000000 	and	x0, x0, x0
