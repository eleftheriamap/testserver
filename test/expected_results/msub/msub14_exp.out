Registers:
X00    = 0000000000e05000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 000000008dab5000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 00000000ff46a000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = fffffffffffffbf1
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x91781400
0x00000004 : 0xd1103fbd
0x00000008 : 0x516e5a10
0x0000000c : 0x1b1dc006
0x00000010 : 0x8a000000
